service/controller/v13/resource/cloudformation/current.go

```diff
@@ -64,6 +64,20 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 
 	var currentState StackState
 	{
+		dockerVolumeResourceName, err := sc.CloudFormation.GetOutputValue(stackOutputs, key.DockerVolumeResourceNameKey)
+		if cloudformationservice.IsOutputNotFound(err) {
+			// Since we are transitioning between versions we will have situations in
+			// which old clusters are updated to new versions and miss the master
+			// instance resource name in the CF stack outputs. We ignore this problem
+			// for now and move on regardless. On the next resync period the output
+			// value will be there, once the cluster got updated.
+			//
+			// TODO remove this condition as soon as all guest clusters in existence
+			// obtain a master instance resource.
+			dockerVolumeResourceName = ""
```

Comment:
Do we want a metric for this being the case? Then, in the future, when we want to make sure that all clusters have moved on, we can add an alert.

Reply:
https://github.com/giantswarm/giantswarm/issues/3682

Reply:
Love it